src/status_im/ui/screens/wallet/send/views.cljs

```diff
@@ -106,18 +107,22 @@
       (i18n/label :t/cancel)]
      [button/button {:style               (wallet.styles/button-container sign-enabled?)
                      :on-press            sign-handler
-                     :disabled?           (or spinning? (not sign-enabled?))
+                     :disabled?           (or spinning?
+                                              (not sign-enabled?)
+                                              (= :offline network-status))
                      :accessibility-label :sign-transaction-button}
       (i18n/label sign-label)
       [vector-icons/icon :icons/forward {:color :white}]]]))
 
 ;; "Sign Transaction >" button
 (defn- sign-transaction-button [amount-error to amount sufficient-funds? sufficient-gas? modal?]
-  (let [sign-enabled? (and (nil? amount-error)
+  (let [online? (= :online @(re-frame/subscribe [:network-status]))
```

Comment:
this looks weird, can we pass online? from outside?

Reply:
@flexsurfer changed, please take a look